pkgs/top-level/stage.nix

```
@@ -75,6 +75,11 @@ let
       inherit (self) runtimeShell;
     };
 
+  testUtilities = self: super:
+    import ../build-support/test-utilities.nix {
+      inherit (self) runCommand;
+    };
```

```suggestion
    super.callPackages ../build-support/test-utilities.nix {};
```

Haven't tested this, but should be lower maintenance. See also next comment.